### DISCUSSION TITLE
Issue forms feedback: access formdata through API

### DISCUSSION THREAD

**Discussion by joraff** [community/community#36162] [Projects and Issues]
I'm beginning to use issue forms, and so far I love where this is going. One thing I wish I could do, however, is access the original form data field values, so I don't have to parse the issue body to find something specific. This could enable automation through actions or webhooks to act on specific field values.

Perhaps appending something like the following the the `issue` object:

```json
{
  "template": {
    "name": "my_issue_template.yaml",
    "form_data": {
      "field_id": {
        "value": "",
        "other field info": ""
      }
    }
  }
}
```

Which could allow use in an action step such as:

```
- name: Assign label based on dropdown value
  run: |
    gh issue edit ${{ github.event.issue.html_url }} --add-label ${{ github.event.issue.template.format_data.field1.value }}
```

**Comment by Timmmy-nlb** [comment]
This would be extremely useful to be able to process data from the form in workflows! 
Or does anyone know another way that would lead to the same goal?
